tabletop/core/view.js

```diff
@@ -50,8 +51,65 @@ View.prototype.drawGridBoard = function() {
   this.animate();
 };
 
-// todo 
-View.prototype.drawPathBoard = function() {
+View.prototype.drawArrayBoard = function() {
+
+  this.boardView.x = c.boardStartX;
+  this.boardView.y = c.boardStartY;
+  this.boardView.beginFill(c.blueColor, 1);
+  this.boardView.drawRect(0, 0, c.boardWidth, c.boardHeight);
+  this.stage.addChild(this.boardView);
+  this.drawArrayTiles();
+  this.drawTokens();
+  this.drawMessage();
+  this.animate();
+
+};
+
+View.prototype.drawArrayTiles = function() {
+  var tileWidth = c.boardWidth / this.game.board.width;
+  var tileHeight = c.boardHeight / this.game.board.height;
+  
+  var tileNum = 0;
+
+  for (var x = 0; x < this.game.board.width; x++) {
```

Comment:
Is there anyway we could abstract some of this? seems like alot of repeated functionaliy. maybe these lines: 
 +     var tileView = this.drawTile(tile, {width: tileWidth, height: tileHeight});
 +    tileView.x = (x * tileWidth);
 +    tileView.y = 0;
 +    this.tileViews[tileNum] = tileView;
 +    this.boardView.addChild(tileView);